codebuild/bin/install_default_dependencies.sh

```diff
@@ -37,6 +37,14 @@ if [[ ("$S2N_LIBCRYPTO" == "openssl-1.1.1") || ("$TESTS" == "integration" || "$T
     fi
 fi
 
+# Download and Install Openssl 3.0
+if [[ "$S2N_LIBCRYPTO" == "openssl-3.0" && ! -d "$OPENSSL_3_0_INSTALL_DIR" ]]; then
+    if [[ ! -x "$OPENSSL_3_0_INSTALL_DIR/bin/openssl" ]]; then
+      mkdir -p "$OPENSSL_3_0_INSTALL_DIR"||true
```

Comment:
Doesn't "-x" check if you have execute permissions on a file? How could we ever fail this check, since the outer if checks that the directory doesn't exist?

You also wouldn't need the "|| true" on the next line, since the directory would always be created.

Also nit: your indentation isn't consistent. It seems off in this entire file tho, so you didn't make it worse.

Reply:
Thats copy pasta for you. Fixed and tested that it works locally. `||true` was present in all the other `mkdir -p` commands but it not necessary from what I can tell so I have removed it.